src/services/user/services/userService.js

```diff
@@ -116,13 +117,15 @@ const userHooks = {
 			iff(isProvider('external'), disallow()),
 			authenticate('jwt'),
 			sanitizeData,
+			checkUniqueEmail,
```

Comment:
Different between 
checkUnique,
checkUniqueAccount,

how the account update work by update method. 

Is it is save that sso accounts can not update, patch her emails?

Reply:
should also check for blockDisposableEmail('email'),